cranelift/src/disasm.rs

```diff
@@ -124,16 +92,14 @@ pub fn print_all(
     print: bool,
     relocs: &[FinalizedMachReloc],
     traps: &[MachTrap],
-    stack_maps: &[MachStackMap],
 ) -> Result<()> {
     print_bytes(&mem);
     print_disassembly(func, isa, &mem[0..code_size as usize])?;
     if print {
         println!(
-            "\n{}\n{}\n{}",
+            "\n{}\n{}",
             print_relocs(&func.params, relocs),
             print_traps(traps),
-            print_stack_maps(stack_maps)
```

Comment:
Can cranelif-tool print user stackmaps?

Reply:
The vcode's debug implementation should include the stack maps:

https://github.com/bytecodealliance/wasmtime/blob/9d35545bf1227c701e37b7c35ec3018ddcdefc43/cranelift/filetests/filetests/isa/x64/user_stack_maps.clif#L55-L68